plugins/vpp/l3plugin/vppcalls/vpp1908/route_dump.go

```diff
@@ -28,82 +27,60 @@ import (
 func (h *RouteHandler) DumpRoutes() ([]*vppcalls.RouteDetails, error) {
 	var routes []*vppcalls.RouteDetails
 	// Dump IPv4 l3 FIB.
-	reqCtx := h.callsChannel.SendMultiRequest(&l3binapi.IPFibDump{})
+	reqCtx := h.callsChannel.SendMultiRequest(&l3binapi.IPRouteDump{})
```

Comment:
As I'm looking at these new API changes in version 3.0.0 of the **ip** module in VPP. This part is not clear to me for use with IPv6. 

Have you tested this with IPv6? Does this dump both IPv4 and IPv6? Because, before we used `IPFibDump`/`IP6FibDump` which both had NO parameters (fields) and with new API we use `IPRouteDump` and there is no IP6RouteDump, but this one takes `Table` parameter which defined as: 

```
// IPTable represents VPP binary API type 'ip_table'.
type IPTable struct {
	TableID uint32
	IsIP6   uint8
	Name    []byte `struc:"[64]byte"`
}
```

And to me it seems like we might need to do extra route dump for IPv6 like: `l3binapi.IPRouteDump{Table: l3binapi.IPTable{IsIP6: true}}`